client/docs/config.json

```diff
@@ -116,6 +131,25 @@
           "Params": "FeegrantParams"
         }
       }
+    },
+    {
+      "url": "./tmp-swagger-gen/cosmos/nft/v1beta1/query.swagger.json",
+      "operationIds": {
+        "rename": {
+          "Params": "NftParams",
+          "Balance": "NftBalance"
+        }
+      }
+    },
```

Comment:
In `provenance`, we'll want to see if we can omit this entry.

Reply:
I was able to exclude all the nft endoints, but couldn't find a way to exclude the related objects. Not having the paths was the key thing though.